aspnetcore/security/data-protection/configuration/scaling.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Configure ASP.NET Core Data Protection in distributed or load-balanced environments
+author: acasey
+description: Learn how to configure Data Protection in ASP.NET Core for multi-instance apps.
+monikerRange: '>= aspnetcore-6.0'
```

Comment:
@Rick-Anderson How is this intended to be used.  Data Protection has been available and scalable for much longer than this, but most of the content in this article is for newer versions.